Users with 0 Created (Teacher) Classrooms cannot access Teacher Classrooms

Issue: If a logged-in user with no created Teacher classrooms accesses https://learn.wildcamgorongosa.org/teachers/classrooms, the app crashes with an infinite spinner.

Analysis:
Error encountered is...

```
action @ 15:59:34.830 RECEIVE_CLASSROOMS 
main-c842746….min.js:34  prev state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action Object {type: "RECEIVE_CLASSROOMS", data: Array[0], error: false, members: undefined}
main-c842746….min.js:34  error TypeError: Cannot read property 'map' of undefined(…)
main-c842746….min.js:34  next state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action @ 15:59:34.836 RECEIVE_CLASSROOMS 
main-c842746….min.js:34  prev state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c842746….min.js:34  action Object {type: "RECEIVE_CLASSROOMS", data: Array[0], error: true}
main-c842746….min.js:34  error TypeError: Cannot read property 'map' of undefined(…)
main-c842746….min.js:34  next state Object {teacher: Object, student: Object, login: Object, users: Object}
main-c8427463beb8d32a6a27.min.js:27 Uncaught (in promise) TypeError: Cannot read property 'map' of undefined(…)o @ main-c842746….min.js:27(anonymous function) @ main-c842746….min.js:34u @ main-c842746….min.js:24(anonymous function) @ main-c842746….min.js:34(anonymous function) @ main-c842746….min.js:34c.dispatch @ main-c842746….min.js:34(anonymous function) @ main-c842746….min.js:22
```

TL;DR of that mess of error messages: the `RECEIVE_CLASSROOMS` action/reducer encounters an undefined when there should be something.

Actions: Investigating.

--- a/src/reducers/teacher.js
+++ b/src/reducers/teacher.js
@@ -24,7 +24,7 @@ export function teacher(state = initialState, action) {
       });
     case types.RECEIVE_CLASSROOMS:
       let uniqueMembers = [];
-      action.members.map((item) => {
+      action.members && action.members.map((item) => {
         if (uniqueMembers.indexOf(item.attributes.zooniverse_login) < 0) {
           uniqueMembers.push(item.attributes.zooniverse_login);
         }

